app/questionnaire/placeholder_parser.py

```diff
@@ -107,7 +110,9 @@ def __call__(
         return self._placeholder_map
 
     def _get_value_source_resolver(
-        self, routing_path_block_ids: OrderedSet[str] | None = None
+        self,
```

Comment:
Since we touch this, can we force named arguments? so:
```
def _get_value_source_resolver(
        self,
        *,
        routing_path_block_ids: OrderedSet[str] | None = None,
        assess_routing_path: bool = False,
    ) -> ValueSourceResolver:
```
which will force you to
```
return self._get_value_source_resolver(
                routing_path_block_ids=OrderedSet(routing_path_block_ids), assess_routing_path=True
            )
```

Reply:
Tried it got a lot of unit test errors will have another look tomorrow morning

Reply:
Update: Forced 👍